### PR TITLE
add topic for "scientific computing"

### PR DESCRIPTION
**Pull request by Mearman**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> This is a popular topic with many repos and deserves additional information.

---

### CODE REVIEW

File: topics/scientific-computing/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Scientific computing
+short_description: Scientific computing is the use of computers to solve scientific problems.
+topic: scientific-computing
+wikipedia_url: https://en.wikipedia.org/wiki/Scientific_computing
```
Comment (smashwilson) [inline topics/scientific-computing/index.md:5]: Looks like this is a redirect, let's just follow it:

```suggestion
wikipedia_url: https://en.wikipedia.org/wiki/Computational_science
```

File: topics/scientific-computing/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+---
+Scientific computing is the use of computers to solve scientific problems. It is a subfield of computational science and computational engineering. It is also a subfield of computer science, in particular computer science that is concerned with the application of algorithms to scientific problems.
```
Comment (smashwilson) [inline topics/scientific-computing/index.md:7]: I know this is a vague topic but I think this description is still a bit too vague... it doesn't actually say anything. Maybe pull in some verbiage from the wikipedia article you've linked?

> It involves the development of models and simulations to understand natural systems.

Comment (GHAZIOMR) [reply topics/scientific-computing/index.md:7]: Mearman:topic/new/scientific-computing

**Comment by kenyonj** [comment]
Please re-open this PR if needed in the future.

**Pull request closed without merging**